playbooks/globalVariables.json

```diff
@@ -1,4 +1,16 @@
 [
+    {
+        "id": 7,
+        "name": "Server_fqhn",
+        "value": "master.fortisoar.in",
```

Comment:
We don't need these global variables as part of this SP

Reply:
Please check with Rohit once

Reply:
@cs-ankit-gupta @cs-suyog-jinturkar "Server_fqhn" global variable can be ignored

Reply:
Ignored means, can be removed. What about other global variables? @cs-rohit-khune

Reply:
Remved all global variable from globalVariables.json